[DECLUTTER BUILDING] separate structure, remove CodeCoverage (#584)
zkSNACKs/WasabiDoc: docs/FAQ/README.md

> This is an archive with some of the many questions that Wasabikas have frequently.

Reviewer: ```suggestion
This is an archive with some of the many questions that Wasabikas have frequently asked, including:
```

> Including about the basics of privacy and Bitcoin in general.

Reviewer: ```suggestion
- Questions about the basics of privacy, and Bitcoin in general.
```

> The important details of how to install Wasabi properly and the best practices for using it.

Reviewer: ```suggestion
- The important details of how to install Wasabi properly and the best practices for using it.
```

> Further, providing guidance for those who are interested in contributing to Wasabi.

Reviewer: ```suggestion
- Further guidance for those who are interested in contributing to Wasabi.
```

> The important details of how to install Wasabi properly and the best practices how to use it.

Reviewer: ```suggestion
The important details of how to install Wasabi properly and the best practices for using it.
```

> Further the questions for those who are interested in contributing to Wasabi.

Reviewer: ```suggestion
Provide guidance for those who are interested in contributing to Wasabi.
```

Author: ```suggestion
Further providing guidance for those who are interested in contributing to Wasabi.
```

Reviewer: Sounds good. Glad you added the needed comma after "Further".
